schema/tables/password_policy.yml

```diff
@@ -0,0 +1,10 @@
+name: password_policy
+examples: >-
+  This policy query will return a 1 if the password policy requires passwords that are 10 characters
+  or longer.
+
+  ```
+
+  SELECT 1 FROM (SELECT cast(lengthtxt as integer(2)) minlength FROM (SELECT SUBSTRING(length, 1, 2) AS lengthtxt FROM (SELECT policy_description, policy_identifier, split(policy_content, '{', 1) AS length FROM password_policy WHERE policy_identifier LIKE '%minLength')) WHERE minlength >= 10);
```

Comment:
This is way more complex that I would imagine. Why is that? 

Can you also split this up so that it's easier to read?